app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -146,36 +164,38 @@ transition.setInterpolator(Interpolator.LINEAR);
 transition.play();
 ```
 
-### Translate Transition
-The [TranslateTransition](javafxdoc:TranslateTransition) creates a move/translate animation that spans its `duration`.
-This is done by updating the `translateX`, `translateY` and `translateZ` properties of the `Node` at regular interval.
+### Pause Transition
+The [PauseTransition](javafxdoc:PauseTransition) executes an `Animation.onFinished` at the end of its execution.
 
-![TranslateTransition](/assets/images/javafx/animation/transition/translate-example.gif)
+### Parallel Transition
+The [ParallelTransition](javafxdoc:ParallelTransition) plays a list of animations in parallel.
```

Comment:
I don't know about using the word "list" here. After all, it uses varargs for these parameters and not a `List`.

Reply:
Well I didn't mention the Java `List` here but instead a list in general.
I still agree that it can cause confusion with the Java `List`, but `list of animations` sounds better in my opinion.
Any suggestions?

Reply:
A group?